.github/workflows/generate.yml

```diff
@@ -4,7 +4,7 @@ name: Generate
 on:
   push:
     branches-ignore:
-      - main
+      - '**'
```

Comment:
Generation has been disabled for now because the updated HTTP client requires a different syntax for call sites in the route clients.

Reply:
https://github.com/seamapi/python-next/pull/51/commits/537fb078886b4689a9dd91f316ce6438122a7a85